### PR TITLE
Removed unnecessary expressions and statements.

### PR DESCRIPTION
**Pull request by Pirulax**

### CODE REVIEW

File: fay.lua [hunk 1]
```diff
@@ -60,45 +60,33 @@ function Sertify.new(testInfo)
...
-			elseif type(self.initialValue) == "table" then
-				evaluation = #self.initialValue == right
-			else
-				evaluation = false
-			end
-			
-			return self:__Report(self.initialValue == nil, testInfo.description)
+		    local evaluation
+		    if type(self.initialValue) == "string" or type(self.initialValue) == "table" then
+			evaluation = #self.initialValue == right
```
Comment (molnarmark) [inline fay.lua:66]: Indentation doesn't match here.

File: fay.lua [hunk 2]
```diff
@@ -60,45 +60,33 @@ function Sertify.new(testInfo)
...
-			elseif type(self.initialValue) == "table" then
-				evaluation = #self.initialValue < right
-			else
-				evaluation = false
-			end
-			
-			return self:__Report(self.initialValue == nil, testInfo.description)		
+		    local evaluation
+		    if type(self.initialValue) == "string" or type(self.initialValue) == "table" then
+			evaluation = #self.initialValue < right
```
Comment (molnarmark) [inline fay.lua:75]: Indentation doesn't match here.

File: fay.lua [hunk 3]
```diff
@@ -60,45 +60,33 @@ function Sertify.new(testInfo)
...
+		    end
+
+		    return self:__Report(self.initialValue, testInfo.description)		
 		end
-		
+
+		self.toBeGreaterThan = function(self, right)
+		    local evaluation
+		    if type(self.initialValue) == "string" or type(self.initialValue) == "table" then
+			evaluation = #self.initialValue > right
```
Comment (molnarmark) [inline fay.lua:84]: Indentation doesn't match here.

**Review by molnarmark — CHANGES_REQUESTED** [review]

**Comment by szekelymilan** [comment]
Why do you always try to find bugs / optimization issues in someone else's code? 
**Do something on your own.** :smile:

**Pull request closed without merging**